Bottom: d52e563764ba9b50ed83e1792954bd39b07b266d
Top:    d6c98d9104a3b62c176ca85af5d0b3ad6a11e610
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-15 13:41:01 +0800

Refresh of tun-introduce-tx-array-mode

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index b1f1612..5276303 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -115,7 +115,7 @@ do {								\
 #define TUN_VNET_BE     0x40000000
 
 #define TUN_FEATURES (IFF_NO_PI | IFF_ONE_QUEUE | IFF_VNET_HDR | \
-		      IFF_MULTI_QUEUE)
+		      IFF_MULTI_QUEUE | IFF_TX_ARRAY)
 #define GOODCOPY_LEN 128
 
 #define FLT_EXACT_COUNT 8
